bin/build_breakpad.sh

```diff
@@ -81,11 +80,10 @@ CFLAGS="${OPTFLAGS}" CXXFLAGS="${OPTFLAGS}" ./configure --prefix="${PREFIX}" ||
 
 CPU_NUM=$(grep ^processor /proc/cpuinfo  | wc -l)
 make -j${CPU_NUM} install
-if [ -z "${SKIP_CHECK}" ]; then
-  #FIXME: get this working again
-  #make check
-  true
-fi
+# if [ -z "${SKIP_CHECK}" ]; then
+#   #FIXME: get this working again
+#   make check
+# fi
```

Comment:
This is a hold-over from when this script was in Mozilla. I'm not sure who wrote that fixme or what's broken here. I commented the whole if block out and we can figure that out later.